app.js

```diff
@@ -13,3 +29,129 @@ function openMenu() {
   menu.style.backgroundColor = '#384af5';
   menu.style.display = 'block';
 }
+
+function popUpProject(popUpdata) {
+  const popUpItem = `<div id="popUp-view">
+     <div class="popUp-card">
+         <img src="images/close.png" alt="close" id="popUp-Close" onclick="popupClose();">
```

Comment:
- [ ] Please change this and all other regular functions to es6 arrow function syntax.

Reply:
Please can you elaborate a bit more on arrow functions as this is not a 1 line function?